### ISSUE TITLE
Outdated release notes link

### ISSUE THREAD

**Issue by leoschwarz** [rust-lang/rust-www#941]
On the page https://www.rust-lang.org/en-US/documentation.html there is the link to `Release Notes` (https://github.com/rust-lang/rust/blob/stable/RELEASES.md) but it has either been forgotten to push the recent updates to this branch (on master there are entries for 1.19 and 1.20) or it's not going to be updated anymore and the link on the web page should be changed to master (will new additions be merged into master only once the version is released as stable?).

**Comment by steveklabnik** [comment]
That always links to the latest release for stable, but sometimes notes
don't make it into the correct stable branch, as we land changes on nightly.

The link should probably point to master instead for this reason.

**Comment by steveklabnik** [comment]
@abitrolly yes, sometimes release notes don't make it into the branch for that release, as I said above.

**Comment by abitrolly** [comment]
Where is the current release procedure documented?

**Comment by steveklabnik** [comment]
https://forge.rust-lang.org/release-process.html

**Comment by abitrolly** [comment]
No step for updating release notes. Why they are on `master` and not even in `beta`?

**Comment by steveklabnik** [comment]
As I mentioned above, new PRs always land in master. We sometimes port things back to `beta`. We only port things to `stable` if we're planning a stable release, which almost never happens.

Because of this, when it's time to think about cutting a release, `beta` has always been branched. This means that the PRs for release notes would have to land on master, and then be backported to beta, and then land there, right in the middle of all the other work for the release.

If the release notes were prepared six weeks ahead of time, then in theory, it would work; but creating release notes is a huge job, and so nobody has wanted to do two releases' worth of notes at the same time, in order to get us ahead so this time schedule would work. Additionally, then the notes would also have to be modified if things land on beta.

Basically, all of this complexity and work hasn't been worth it yet, as you can always check `master` to see the notes anyway. Yes, it would be nice, but there's always so much work to do. We do accept help in making the release notes, so if this is something you care deeply about, putting in the work to fix it would be great! happy to walk you through that process if you're interested.

**Comment by abitrolly** [comment]
Let's wait for the problem to manifest itself again. Every hit increases the chance to make this usability/process issue fixed. )
